admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -709,7 +712,7 @@ def _schemaForCSS(self, database, table):
         schema = self.czarWmgr.tableSchema(database, table)
         # strip CREATE TABLE
         i = schema.find('(')
-        return schema[i:]
+        return schema[i:].encode('utf_8')
```

Comment:
Because it's a big unicode mess :cry: CssAccess cannot accept unicode, but wmgr (being a web service client) returns unicode strings, so this string needs to be encoded at some point.  I could do it at a point where I call CssAccess or here. No other strings in the module are unicode, so I just say I better drop unicode completely and do it here.